docs/features/multiple-networks/multi-network-policies.md

```diff
@@ -1,30 +1,55 @@
 # MultiNetworkPolicies
-OVN-Kubernetes implements native support for
-[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy),
-an API providing
+
+## Introduction
+
+[multi-networkpolicy](https://github.com/k8snetworkplumbingwg/multi-networkpolicy) provides
 [network policy](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
-features for secondary networks.
+features for secondary networks, and allow enhanced traffic security.
+
+## Motivation
+
+Using traffic restrictions based on user network configuration is a recommended practice. By defining policies that
+specify which services a workload can access, we mitigate supply chain attacks. Even if a component is compromised, it
+won’t be able to jeopardize other components of the system.
+
+## User-Stories
+
+As a cluster administrator, I want to enforce non-overridable admin network policies that must be adhered to by all user
```

Comment:
removed "admin".

Reply:
just removing the word admin is not enough, since you're still describing something that applies across the entire cluster (for all tenants). 

Network policies are a namespace scoped resource.